Gestao-Energia/Front-end
# Fix login layout

Task:
Merge fix-login-layout


I created the login screen layout based on figma
[Design layout](https://www.figma.com/design/8jD1yTW5lkpWANIlFAUt5v/Telas-SGE?node-id=173-53&t=2ZlXjjQJFh8vSIwd-4)

How To test:

login and password form and confirm button

OBS
The form is not sending any data to api
